ecosystem/runners/runner.py

```diff
@@ -43,6 +47,39 @@ def tear_down(self):
         if self.cloned_repo_directory and os.path.exists(self.cloned_repo_directory):
             shutil.rmtree(self.cloned_repo_directory)
 
+    def get_config(self, file, file_fault, ecosystem_deps: Optional = None):
```

Comment:
Forgot about typing
```python
def get_config(self, file: List[str], file_fault: List[str], ecosystem_deps: Optional[List[str]] = None):
```
with typing you can see that `file` is confusing name since it's not a file it's a list of file names :)